source/blog/2016-04-08-new-in-rdo-repos-one-size-doesn-t-fit-all.html.md

```diff
@@ -32,21 +32,22 @@ This is probably what you want to use for your stable cloud with a community-sup
 
 
 ### RDO Trunk repos
-Provide RPM package repositories for OpenStack projects as close as possible to the vanilla upstream code in order to test any change and catch any bug or impact on packaging as soon as possible. These packages are built when a commit is merged in upstream master or any of the supported stable branches. This means that new packages are provided even during the development phase of a new release (before GA). Once a build is done, a set of CI jobs is passed to validate installation and basic functionalities. As part of RDO Trunk we provide different repos for CentOS 7 under http://buildlogs.centos.org/centos/7/cloud/x86_64/ :
+Provide RPM package repositories for OpenStack projects as close as possible to the vanilla upstream code in order to test any change and catch any bug or impact on packaging as soon as possible. These packages are built when a commit is merged in upstream master or any of the supported stable branches. This means that new packages are provided even during the development phase of a new release (before GA). Once a build is done, a set of CI jobs is passed to validate installation and basic functionalities. As part of RDO Trunk we provide different repos for CentOS 7 under http://trunk.rdoproject.org/centos7-&lt;release>/ :
 
-- **rdo-trunk-&lt;branch>:** provides the latest snapshot of packages successfully built (i.e. rdo-trunk-mitaka).
-- **rdo-trunk-&lt;branch>-tested:** provides the latest snapshot of packages built and with CI jobs successfully passed (i.e. rdo-trunk-master-tested).
-- **rdo-trunk-master-tripleo:** provides the latest snapshot of packages tested by specific TripleO CI jobs on master branch.
+- **consistent:** provides the latest snapshot of packages successfully built (i.e. https://trunk.rdoproject.org/centos7-master/consistent/).
+- **current-passed-ci:** provides the latest snapshot of packages built and with CI jobs successfully passed (i.e. https://trunk.rdoproject.org/centos7-master/current-passed-ci/).
+- **current-tripleo:** provides the latest snapshot of packages tested by specific TripleO CI jobs on master branch (https://trunk.rdoproject.org/centos7-master/current-tripleo/)
 
-Currently (apr '16'), &lt;branch> can be master, mitaka, liberty or kilo.
+Currently (may '16'), &lt;release> can be master, mitaka or liberty.
```

Comment:
This <release> refers to the url http://trunk.rdoproject.org/centos7-<release>/ and there, we use lowercase so i think it's better to leave it as is.